lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -160,7 +182,9 @@ class LighthouseViewerReport {
     return this.github.createGist(this.json).then(id => {
       ga('send', 'event', 'report', 'created');
 
+      this.disableShareButton();
```

Comment:
I can add that if you feel strongly about it. We do already have a visual message something is happening ("saving gist"), so I'm not too worried about someone hitting the button several times.